.gitmodules

```diff
@@ -62,3 +62,6 @@
 [submodule "metagraph/external-libraries/Simple-Web-Server"]
 	path = metagraph/external-libraries/Simple-Web-Server
 	url = https://gitlab.com/eidheim/Simple-Web-Server.git
+[submodule "metagraph/external-libraries/gfapy"]
```

Comment:
Is this enough for making git download this gfapy when cloning/refreshing the repo? No "gfapy" is listed at this path: https://github.com/ratschlab/metagraph/tree/gfa_integration_tests/metagraph/external-libraries